src/collector/collector.ts

```diff
@@ -50,6 +52,10 @@ const jobs = [
   {
     method: proposalCollector.run.bind(proposalCollector),
     cron: '30 * * * * *'
+  },
+  {
+    method: dashboardCollector.run.bind(dashboardCollector),
+    cron: '5 1 * * * *'
```

Comment:
why is this 5 1 instead of 0 0?

Reply:
just to make sure it wont miss any data at 0 0 as it uses others collector data. We can move use 0 30 too.

Reply:
ok than let's change it to 0 1